aws/resource_aws_ecs_service.go

```diff
@@ -204,6 +204,24 @@ func resourceAwsEcsService() *schema.Resource {
 					},
 				},
 			},
+
+			"service_registries": {
```

Comment:
I think this can only support `MaxItems == 1`.  When I tried using more than one registry in #4026 I got back an error from AWS.

Reply:
Thank you!